config/webpack.js

```diff
@@ -26,7 +27,7 @@ if (customConfig && customConfig.webpack) {
   custom2 = customConfig.webpack
 }
 
-const specific = _.defaultsDeep({}, custom1, custom2)
+const specific = merge(custom1, custom2)
```

Comment:
see above, one is custom configs from `package.json` and the other from `.aegir.js`